lib/utils/constants.dart

```diff
@@ -60,6 +60,8 @@ const kWalletAndroidId = 'tech.pylons.wallet';
 const kWalletWebLink = 'https://wallet.pylons.tech';
 const kWalletDynamicLink = 'pylons.page.link';
 
+const kEaselEmail = "easel@pylons.tech";
```

Comment:
Does this email exist or it needs to be created?

Reply:
I am not sure about this because it was in the design.